spark/src/main/scala/io/delta/sharing/spark/DeltaSharingClient.scala

```diff
@@ -412,14 +424,15 @@ private[spark] class DeltaSharingRestClient(
       }
     }
 
-  // Append SparkStructuredStreaming in the USER_AGENT header, in order for the delta sharing server
+  // Add SparkStructuredStreaming in the USER_AGENT header, in order for the delta sharing server
   // to recognize the request for streaming, and take corresponding actions.
   private def getUserAgent(): String = {
-    DeltaSharingRestClient.USER_AGENT + (if (forStreaming) {
-      s" ${DeltaSharingRestClient.SPARK_STRUCTURED_STREAMING}/$STREAMING_VERSION"
+    val sparkAgent = if (forStreaming) {
```

Comment:
Why is this still needed? Is it to track the streaming requests on the provider side?

Reply:
yes. 1. tracking in the usage log 2. gate traffic during private preview.